changelogs/unreleased/1670-mittachaitu

```diff
@@ -0,0 +1 @@
+feat(volume replica): enable REBUILD_ESTIMATES feature gate by default from OpenEBS 1.10.0 version
```

Comment:
Can we use the `( )` to mention which replica - cstor or jiva

Reply:
Updated